### PR TITLE
Complete the homework №5

### PR DESCRIPTION
**Pull request by natak8200**

### CODE REVIEW

**Comment by AntBelyi** [comment]
is title correct?
Screenshot_4

**Comment by natak8200** [comment]
Correct title homework 5

пн, 15 июл. 2024 г., 13:12 AntBelyi ***@***.***>:

File: src/main/java/hw_5/myClass/MyInt.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package hw_5.myClass;
+
+public class MyInt {
+    public int x;
+
+    public MyInt(int x){
+        this.x = x;
+    }
+    @Override
```
Comment (AntBelyi) [inline src/main/java/hw_5/myClass/MyInt.java:10]: Please add extra space between 8 and 9 rows

File: src/main/java/hw_5/myClass/Point.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+public class Point {
+    public int x;
+    public int y;
+
+    public Point(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+    public String Info(){
```
Comment (AntBelyi) [inline src/main/java/hw_5/myClass/Point.java]: method name should starts from the small letter

File: src/main/java/hw_5/myCity/Main.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    public static void main(String[] args) {
+
+        Point[] points = new Point[4];
+        points[0] = new Point(2, 8);
+        points[1] = new Point(5, 1);
+        points[2] = new Point(10, 4);
+        points[3] = new Point(9, 4);
+
+        for (int i = 0; i < points.length; i++) {
+            points[i].Info();
```
Comment (AntBelyi) [inline src/main/java/hw_5/myCity/Main.java]: For now this line is doing nothing. Why do you need it? If you want this line display something in console then fix it

File: src/main/java/hw_5/myCity/Main.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            System.out.println(points[i]);
```
Comment (AntBelyi) [inline src/main/java/hw_5/myCity/Main.java]: Are you sure you need this line?

File: src/main/java/hw_5/myCity/Main.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+            if (points[i].x % 2 == 0 && points[i].y % 2 == 0) {
+                points[i].Info();
```
Comment (AntBelyi) [inline src/main/java/hw_5/myCity/Main.java]: For now this line is doing nothing. Why do you need it? If you want this line display something in console then fix it

File: src/main/java/hw_5/myCity/Main.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+                System.out.println(points[i]);
```
Comment (AntBelyi) [inline src/main/java/hw_5/myCity/Main.java]: Are you sure you need this line?

File: src/main/java/hw_5/myClass/MyInt.java (hunk 1 above)
Comment (natak8200) [reply src/main/java/hw_5/myClass/MyInt.java:10]: fixed

File: src/main/java/hw_5/myClass/Point.java (hunk 2 above)
Comment (natak8200) [reply src/main/java/hw_5/myClass/Point.java]: fixed

File: src/main/java/hw_5/myCity/Main.java (hunk 3 above)
Comment (natak8200) [reply src/main/java/hw_5/myCity/Main.java]: fixed

File: src/main/java/hw_5/myCity/Main.java (hunk 4 above)
Comment (natak8200) [reply src/main/java/hw_5/myCity/Main.java]: fixed

File: src/main/java/hw_5/myCity/Main.java (hunk 5 above)
Comment (natak8200) [reply src/main/java/hw_5/myCity/Main.java]: fixed

File: src/main/java/hw_5/myCity/Main.java (hunk 6 above)
Comment (natak8200) [reply src/main/java/hw_5/myCity/Main.java]: fixed

**Comment by natak8200** [comment]
fixed

**Pull request merged**